common/flatpak-utils.c

```diff
@@ -5955,6 +5948,36 @@ flatpak_repo_generate_appstream (OstreeRepo   *repo,
   return TRUE;
 }
 
+void
+flatpak_appstream_get_xml_path (GFile        *root,
+                                GFile       **appstream_file_out,
+                                GFile       **app_info_dir_out,
+                                const char   *name,
+                                GCancellable *cancellable)
+{
+  g_autoptr(GFile) appstream_file = NULL;
+  g_autoptr(GFile) app_info_dir = NULL;
+
+  appstream_file = g_file_resolve_relative_path (root, "files/share/swcatalog/xml/flatpak.xml.gz");
+  if (g_file_query_exists (appstream_file, cancellable))
+    app_info_dir = g_file_resolve_relative_path (root, "files/share/swcatalog");
+  {
```

Comment:
Did you mean else here? Also use brackets around both parts if so.

Reply:
I believe I did. add an else doesn't effect anything but it makes sense looking at the code